api/v1alpha1/openapi.yaml

```diff
@@ -1858,6 +1859,9 @@ components:
         image:
           type: string
           description: Image of the container.
+        engine:
+          type: string
```

Comment:
since this is declared why do we need to define it per container? If we must have an engine type I would expect the actual containers to be inside of it.
```
status:
  workload:
    -  engine: Podman
      <workload status>
```

Reply:
We can revist down the road I guess.

Reply:
You could have some running as part of crio, and some running as part of podman. 

We could may be optimize by having two different arrays?

Reply:
right, that is what I described above an array of engines each with its own containers vs each container describing its engine